_rules/image-non-empty-accessible-name-23a2a8.md

```diff
@@ -214,24 +216,36 @@ This element has a [semantic role][] of `img`, but is not [included in the acces
 
 #### Inapplicable Example 3
 
-This `img` element is not [included in the accessibility tree][].
+This `img` element is hidden with `aria-hidden` set to "true".
 
 ```html
 <img src="/test-assets/shared/w3c-logo.png" aria-hidden="true" />
 ```
 
 #### Inapplicable Example 4
 
-This element is neither an `img` element nor has a role of `img`.
+This `img` element is hidden because its parent has `display: none`.
 
 ```html
-<div aria-label="W3C logo"></div>
+<div style="display: none">
+	<img src="/test-assets/shared/w3c-logo.png" />
+</div>
+```
+
+#### Inapplicable Example 5
+
+This `img` element is hidden with `visibility: hidden`.
```

Comment:
```suggestion
This `img` element is hidden because its parent has `visibility: hidden`.
```

Reply:
I think that's an unnecessary detail that is obvious from the example itself.